dashboard/src/main/kotlin/com/ritense/dashboard/repository/DashboardRepository.kt

```diff
@@ -21,4 +21,11 @@ import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.stereotype.Repository
 
 @Repository
-interface DashboardRepository : JpaRepository<Dashboard, String>
+interface DashboardRepository : JpaRepository<Dashboard, String> {
+
+    fun findAllByOrderByOrder(): List<Dashboard>
```

Comment:
ByOrderByOrder? Is that correct?

Reply:
I've just tried it and unfortunately that doesn't work. It must start with `findAllBy`